README.md

```diff
@@ -1,5 +1,5 @@
 This here's a tail for all the fellas
-Tryin; to do what those ladies tell us
```

Comment:
I'm guessing the semi-colon was meant to be an apostrophe in the first place. If this PR is declined, be sure to change **Tryin;** to **Tryin'**.